xml/docu_styleguide.language.xml

```diff
@@ -63,20 +63,20 @@
     occurrences of the term in this chapter should then use the acronym.
    </para>
    <para>
-    Create plural forms of acronyms by adding a lowercase <quote>s</quote>.
+    Create plural forms of acronyms by adding a lowercase <quote>s.</quote>
     For example, use <quote>CDs</quote> and <quote>BIOSes.</quote> Never add
     an apostrophe before the <quote>s</quote> or <quote>es.</quote>
    </para>
    <para>
     For clarity, avoid using possessive forms of acronyms. For example, do not
-    use <quote>XMLʼs specification</quote>.
+    use <quote>XMLʼs specification.</quote>
    </para>
   </sect3>
   <sect3 xml:id="sec-latin">
    <title>Latin abbreviations</title>
    <para>
     Do not use Latin abbreviations. Use the full English form: for example,
-    use <quote>that is</quote> instead of <quote>i.e.</quote>. As an
+    instead of <quote>i.e.,</quote> use <quote>that is</quote>. As an
```

Comment:
I think this change makes it harder to understand and again introduces the problem of punctuation within quotation marks. What was so bad about the old text? I suggest to stick to it.
```suggestion
    use <quote>that is</quote> instead of <quote>i.e.</quote>. As an
```

Reply:
So by the AE rules, this should be "i.e.," but I agree to keep the old wording here to avoid a full stop mess at the end of the sentence :-)